perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/bash
 # It will take approx 80 minutes to run the script.
-set -e
 sudo apt-get update
```

Comment:
We can move this also below to perf-test-check.

Reply:
fixed